Major doc update para seller behavior (#1258)
mercadopago/devsite-docs: guides/plugins/woocommerce/integration.en.md

Once the module is installed, follow these steps to integrate:

1. Create a [seller account](https://www.mercadopago[FAKER][URL][DOMAIN]/registration-company?confirmation_url=https%3A%2F%2Fwww.mercadopago[FAKER][URL][DOMAIN]%2Fcomo-cobrar) in Mercado Pago if you don’t have one yet.
2. Obtain the Access Token and Public Key [credencials](/https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/credentials/credentials) and paste them into the Production and Testing fields that you will find in the module configuration.
3. Approve the account to [go to production](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/online-payments/checkout-api/goto-production) and receive money from your sales in Mercado Pago.

Done! With this you can capture the payments you receive in WooCommerce with your Mercado Pago account.

![Credentials flow](/images/woocomerce/es_woo_credenciales.gif)

> WARNING
>
> Important
>

> > The [credentials](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/credentials/credentials) are the keys that uniquely identify you within the platform. To operate in a **test environment** with which to simulate online payments, use the test credentials. Use **production** credentials to receive real payments.

Reviewer: ```suggestion
> The [credentials](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/credentials/credentials) are the keys that uniquely identify you within the platform. To operate in a **test environment** in order to simulate online payments, use the test credentials. To receive real payments, use **production** credentials.
```
